.github/workflows/pr-changelog-check.yml

```diff
@@ -11,5 +11,5 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Changelog check
-      if: ${{ !contains(github.event.pull_request.labels.*.name, 'changelog-not-required') }}
+      if: ${{ !contains(github.event.pull_request.labels.*.name, 'changelog-not-required') || !contains(github.event.pull_request.labels.*.name, 'Design') || !contains(github.event.pull_request.labels.*.name, 'Website') || !contains(github.event.pull_request.labels.*.name, 'Documentation')}}
```

Comment:
Looks like this broke the logic for the check since it doesn't have all the labels... worked too fast. Need to double check my logic.